features/apim-analytics-feature/org.wso2.analytics.solutions.apim.analytics.feature/src/main/resources/siddhi-files/APIM_ACCESS_SUMMARY.siddhi

```diff
@@ -112,4 +127,4 @@ select apiContext, apiName, apiVersion, apiCreator,apiCreatorTenantDomain, apiHo
 update or insert into ApiLastAccessSummary
 set ApiLastAccessSummary.apiContext = apiContext, ApiLastAccessSummary.apiVersion = apiVersion, ApiLastAccessSummary.applicationOwner = applicationOwner, ApiLastAccessSummary.tenantDomain = tenantDomain, ApiLastAccessSummary.lastAccessTime = lastAccessTime
 on ApiLastAccessSummary.apiName == apiName and ApiLastAccessSummary.apiCreatorTenantDomain == apiCreatorTenantDomain and ApiLastAccessSummary.apiHostname == apiHostname and
-    ApiLastAccessSummary.gatewayType == gatewayType and ApiLastAccessSummary.label == label;
+    ApiLastAccessSummary.gatewayType == gatewayType and ApiLastAccessSummary.label == label;
```

Comment:
add new line

Reply:
added in 389b8295bf177fa698150dd87a9014ded5aa64a8